docs/components/component_spec.md

```diff
@@ -19,14 +19,11 @@ name: ...
 description: ...
 image: ...
 
-consumes:
-  ...
+consumes: ...
 
-produces:
-  ...
+produces: ...
 
-args:
-  ...
+args: ...
```

Comment:
Don't know why this was changed, but they are meant to be on the line below as that's where the actual values will be.

Reply:
Yeah sorry about that... It's due to an extension I use that "cleans" the code when I save it. I'm disabling it!